client/src/resources/lang/en_us.json

```diff
@@ -463,131 +463,131 @@
     "chatMessage.trapState.dismantled": "Your trap is currently dismantled.",
     "chatMessage.trapState.ready": "Your trap is currently ready.",
     "chatMessage.trapState.set": "Your trap is currently set.",
-    "chatMessage.vigilanteSuicide":"You committed suicide over the guilt of killing an innocent person.",
+    "chatMessage.vigilanteSuicide":"You committed suicide over the guilt of killing an innocent townie.",
```

Comment:
probably remove the word innocent and change it to townie, since you did this elsewhere